src/services/utilities.ts

```diff
@@ -475,11 +500,13 @@ namespace ts {
         }
 
         if (node) {
-            let jsDocComment = node.jsDocComment;
-            if (jsDocComment) {
-                for (let tag of jsDocComment.tags) {
-                    if (tag.pos <= position && position <= tag.end) {
-                        return tag;
+            let jsDocComments = node.jsDocComments;
+            if (jsDocComments) {

```

Comment:
You should either assign to this new variable, `jsDocComments` or do what use this way: "https://github.com/Microsoft/TypeScript/pull/8103/files#diff-233e1126c0abc811c4098757f9e4516eR7807" ?